### ISSUE TITLE
DevTest: support for the `2018-10-15-preview` swagger

### ISSUE THREAD

**Issue by tombuildsstuff** [Azure/azure-rest-api-specs#4105]
👋 

I'm trying to add support for DevTestLabs to Terraform - but have noticed the latest API being used in the Azure Portal isn't available via the Azure SDK for Go, which upon further investigation is because the Swagger is missing in this repository.

Would it be possible to add the Swagger for version `2018-10-15-preview` of the DevTestLabs API's?

Thanks!

cc @joshgav @achandmsft

**Comment by jviau** [comment]
Hi @tombuildsstuff , I am currently in the process of GA'ing a new version of DevTestLabs. It won't be `2018-10-15-preview` exactly, but it will be identical.

**Comment by tombuildsstuff** [comment]
@jviau cool thanks. No worries if the date varies slightly between Preview and GA - I'm mostly trying to determine what the future behaviours will be, given there's currently substantial differences between the Portal and the old API version which make it hard to determine exactly what we should be supporting :)

Thanks!

**Comment by tombuildsstuff** [comment]
@jviau just to follow up here, I've noticed that [`2018-09-15` is now available in the tree](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2018-09-15) - is this the version you were referring too?

**Comment by jviau** [comment]
@tombuildsstuff yes it is available now on NuGet: https://www.nuget.org/packages/Microsoft.Azure.Management.DevTestLabs/3.0.0. Sorry I didn't update this, I was waiting on a blog post to share with you.

**Comment by tombuildsstuff** [comment]
@jviau cool thanks - we use the Go SDK rather than the .net clients - so I'm going to ask @jhendrixMSFT / @joshgav if it'd be possible to update the codegen so this is available in the SDK?

Thanks!

**Comment by tombuildsstuff** [comment]
cool, thanks @jhendrixMSFT :)
